CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Added
+- `pwrite` hook (used by `dotnet`);
```

Comment:
Awesome! Can you adjust the tests to be integration instead of e2e? all tests that check layer functionality should be integration now that we have the framework/boilerplate.